src/sentry/tasks/reprocessing2.py

```diff
@@ -185,92 +178,3 @@ def finish_reprocessing(project_id, group_id):
     from sentry import similarity
 
     similarity.delete(None, group)
-
-
-def _json_size(*json_blobs):
-    from sentry.nodestore.base import json_dumps
-
-    bytestring = b"\n".join(json_dumps(data).encode("utf8") for data in json_blobs)
-    cctx = zstandard.ZstdCompressor()
-    return len(cctx.compress(bytestring))
-
-
-@instrumented_task(name="sentry.tasks.reprocessing2.capture_nodestore_stats")
-def capture_nodestore_stats(cache_key, project_id, event_id):
-    set_current_project(project_id)
-
-    from sentry.eventstore.compressor import deduplicate
-    from sentry.eventstore.models import Event
-
-    node_id = Event.generate_node_id(project_id, event_id)
-    data = nodestore.get(node_id)
-
-    if not data:
-        metrics.incr("eventstore.compressor.error", tags={"reason": "no_data"})
-        return
-
-    old_event_size = _json_size(data)
-
-    unprocessed_data = event_processing_store.get(_get_unprocessed_key(cache_key))
-    event_processing_store.delete_by_key(_get_unprocessed_key(cache_key))
```

Comment:
Where will we delete the unprocessed payload now?

Reply:
The subkey is deleted along with the main payload since it's part of the same nodestore entry.

Reply:
michal is right, there's a leak. this is redis, it's just called "store"

Reply:
oh right, it's before we go into nodestore. sorry about that.